qcodes/tests/test_config.py

```diff
@@ -152,39 +153,39 @@ def default_config(user_config: Optional[str] = None):
     Args:
         user_config: represents the user config file content.
     """
-    home_file_name = qcodes.Config.home_file_name
-    schema_home_file_name = qcodes.Config.schema_home_file_name
-    env_file_name = qcodes.Config.env_file_name
-    schema_env_file_name = qcodes.Config.schema_env_file_name
-    cwd_file_name = qcodes.Config.cwd_file_name
-    schema_cwd_file_name = qcodes.Config.schema_cwd_file_name
-
-    qcodes.Config.home_file_name = ''
+    home_file_name = qcodes.configuration.Config.home_file_name
```

Comment:
is `qcodes.configuration.` prefix needed if `Config` class has alrady been imported into this namespace?